src/main/java/com/creativetechguy/NoBadAlchsPlugin.java

```diff
@@ -50,22 +51,35 @@ public class NoBadAlchsPlugin extends Plugin {
             ItemID.EMERALD_6896,
             ItemID.RUNE_LONGSWORD_6897
     );
+    private List<String> allowList = new CopyOnWriteArrayList<>();
```

Comment:
I don't know whether this is required or not, I saw it used somewhere else so I figured better safe than sorry. If  unnecessary it can be removed.